langwatch_nlp/langwatch_nlp/studio/utils.py

```
@@ -297,3 +303,75 @@ def default(self, o):
             return super().default(o)
         except:
             return str(o)
+
+
+def get_dataset(
```

Pylint (3.3.7)

[refactor] Too many local variables (16/15)

(R0914)